### PR TITLE
Challenge

### PR DESCRIPTION
**Pull request by cwilkosz**

### CODE REVIEW

**Comment by cwilkosz** [comment]
finished

File: index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+<head>
+	<meta charset="utf-8">
+	 <!--Header element `<h1>` - This is the main headline; include your name and what this page is-->
+	 
+
+
+	 <link rel="stylesheet" type="text/css" href="styles.css">
+	<!--Title - make it meaningful--> 
+	<title>Craig Wilkosz HTML Quiz</title>
+	<h1>Craig Wilkosz HTML meaningful Quiz...</h1>
```
Comment (morecallan) [inline index.html]: This h1 tag should be in the body. (Maybe in a header element)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    </nav>
+
+<!--Article element containing your bio-->
+	<header>
+    <h2>About Craig</h2>
+  </header>
+	  <section>
+	  	 <article>
+
+        <title>About Craig</title>
```
Comment (morecallan) [inline index.html]: Instead of using title tag here (it should only be in the head of an HTML), wrap these in a header tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+		</article>
+
+		<div>
+	  		<h3>Here lies a link to a fun place to visit on the web and in real life...</h3>
+			<a href="url">http://www.tnrenfest.com/index.htm</a>
+	  		<!-- Article element with links to resources or sites you like-->	
+  		</div>
+
+  		<article>
```
Comment (morecallan) [inline index.html]: Take this comment from the specs literally "Article element with two sections for future projects (placeholders)" - (wink, use sections, wink).

**Review by morecallan — COMMENTED** [review]
@cwilkosz Great start! I like that you are making good use of commenting your code with the project specs to make it like a "checklist." You also have a great commit history and I am digging the use of q tags. 

For your next submission, check the notes I left directly on the code and also take some time to check the indentation of your elements.

Nice work so far!

**Comment by cwilkosz** [comment]
round 2
@morecallan

**Comment by morecallan** [comment]
Very nicely done! 
👍

**Pull request merged**